### PR TITLE
Ohie 115

### PR DESCRIPTION
**Pull request by nthfloor2**
@hnnesv The console is successfully hitting the API and manually triggering the channel. However I am still getting a 404 error on the browser. Still looking into it.

If you spot anything obvious or that needs that needs to be reviewed, let me know.

### CODE REVIEW

**Comment by hnnesv** [comment]
Ok cool, thanks @nthfloor2! I think it'd be best if @rcrichton picks this up as he's the owner of this story.

File: docs/dev-guide/api-ref.md [hunk 1]
```diff
@@ -101,6 +101,12 @@ where `:channelId` is the `_id` property of the channel to delete.
 
 The response code will be `200` if successful.
 
+#### Manually Trigger Polling Channel
+
+'GET /channels/:channelId/trigger'
```
Comment (rcrichton) [inline docs/dev-guide/api-ref.md]: I think this needs to be a `POST` as `GET` requests shouldn't have any side affects if hit multiple times. Perhaps we should switch it over?

File: src/api/channels.coffee [hunk 2]
```diff
@@ -225,3 +228,53 @@ exports.removeChannel = (channelId) ->
...
+
+###
+# Manually Triggers Polling Channel
+###
+exports.triggerChannel = (channelId) ->
+
+  # Test if the user is authorised
+  # if authorisation.inGroup('admin', this.authenticated) is false
+  #   utils.logAndSetResponse this, 403, "User #{this.authenticated.email} is not an admin, API access to removeChannel denied.", 'info'
+  #   return
```
Comment (rcrichton) [inline src/api/channels.coffee]: The above code shouldn't be commented out.

Comment (rcrichton) [inline src/api/channels.coffee]: Oh, I see you are doing this check below. It should be able to be done here rather similar to the other endpoints code.

File: src/api/channels.coffee [hunk 3]
```diff
@@ -225,3 +228,53 @@ exports.removeChannel = (channelId) ->
...
+
+  # Get the values to use
+  id = unescape channelId
+
+  # need to Initialize return status otherwise will always return 404
+  this.status = 200
+
+  try
+    channel = null
+    accessDenied = false
```
Comment (rcrichton) [inline src/api/channels.coffee]: I don't think this var is used.

**Comment by rcrichton** [comment]
@nthfloor2 Thanks! this looks good. Just a few comments above.

**Comment by nthfloor2** [comment]
@rcrichton thank you for the comments, have updated the code - please review. I am seeing that some of the integration tests are failing - but they are passing locally. Is that normal, or am I missing something?

**Comment by rcrichton** [comment]
Cool looks good to me. Ya, the test are failing intermittently on Travis atm. We are yet to figure this out. I'l rerun the tests and merge in once they are passing.

**Comment by rcrichton** [comment]
:+1:

**Pull request merged**